### PR TITLE
platform dependent test change

### PR DESCRIPTION
**Pull request by rich-bugsnag**

### CODE REVIEW

File: test/desktop/features/handled_errors.feature [hunk 1]
```diff
@@ -25,7 +25,12 @@ Feature: Handled Errors and Exceptions
         And the event "device.runtimeVersions.unityScriptingBackend" is not null
         And the event "device.runtimeVersions.dotnetScriptingRuntime" is not null
         And the event "device.runtimeVersions.dotnetApiCompatibility" is not null
-        And the event "app.type" equals "Mac OS"
+        And the event "app.type" equals the platform-dependent string:
+        | macos | "Mac OS" |
```
Comment (twometresteve) [inline test/desktop/features/handled_errors.feature:29]: Needs to be without the `"`s.  Also lose the blank lines below.

Comment (twometresteve) [reply test/desktop/features/handled_errors.feature:29]: Also, while I'm being pedantic, please indent line 29/30 to align with other similar steps.

Comment (rich-bugsnag) [reply test/desktop/features/handled_errors.feature:29]: done

**Comment by twometresteve** [comment]
This PR is no longer needed - overtaken by other changes.

**Pull request closed without merging**